pkg/frontend/transport/handler.go

```diff
@@ -252,6 +252,52 @@ func (f *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (f *Handler) parseURLValues(r *http.Request, stats *querier_stats.QueryStats, values url.Values) {
```

Comment:
Should we add a unit test for this function?

Reply:
Will add the test func